bin/vaultclient

```diff
@@ -190,6 +195,21 @@ program
         }, handleVaultResponse);
     }));
 
+program
+    .command('get-accounts')
+    .option('--account-ids <ACCOUNT ID,ACCOUNT ID,...>')
+    .option('--canonical-ids <CANONICAL ID,CANONICAL ID,...>')
+    .option('--emails <EMAIL,EMAIL,...>')
+    .action(action.bind(null, 'get-accounts', (client, args) => {
+        client.enableIAMOnAdminRoutes().getAccounts(
+            args.accountIds ? args.accountIds.split(',') : null,
```

Comment:
Given we are in node 16 now, I think you can do optional chaining here.
```suggestion
            args.accountIds?.split(',') || null,
```

Reply:
Sweet!